### ISSUE TITLE
The validation-set loss of the transformer-lm in recipe librispeech gradually increase！！！

### ISSUE THREAD

**Issue by YeBobr** [espnet/espnet#2512]
Hello,
    The training curves of the transformer language model are as follows when I use the recipe librispeech. Does anyone know what the problem is？
![IMG_140](https://user-images.githubusercontent.com/18595598/94218920-226ba780-ff18-11ea-847a-d6f432611f1f.JPEG)

**Comment by sw005320** [comment]
Did you use the 4 GPUs? Some hyper-parameters are optimized for the 4GPU setup as shown in
https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/conf/tuning/lm_transformer.yaml#L1-L4

**Comment by YeBobr** [comment]
Yes, I use the 4 GPUs with 11G memory， and I use the hyper-parameters in  https://github.com/espnet/espnet/blob/master/egs/librispeech/asr1/conf/tuning/lm_transformer.yaml. I still have no idea to train the language model better.

**Comment by ShigekiKarita** [comment]
When I ran the same config with pytorch 1.5.0 and 5 gpus in the last month (commit a753ba4f57a6d72336974849503871f3ff9ff7e8), it was OK. What was your `val_perplexity`? Here is mine
```
$ grep val_perplexity exp/train_rnnlm_pytorch_lm_unigram5000_ngpu4/log      
        "val_perplexity": 41.20009502348011
        "val_perplexity": 36.742481484193426
        "val_perplexity": 34.6047906231087
```

The final val_perplexity was 30.6 when I uploaded the model https://github.com/espnet/espnet/pull/1246#issuecomment-541431024

**Comment by YeBobr** [comment]
Thanks a lot. 
I ran the config with pytorch 1.1 with espent 0.9.0  and 4 gpus. I didn't use the external data to train the language model. My last three val_perplexitys are as follows:
        "val_perplexity": 1028.6793601629045
        "val_perplexity": 1113.3659281667658
        "val_perplexity": 1183.5428165710855
